### ISSUE TITLE
Slack Invite URL update?

### ISSUE THREAD

**Issue by mattzollinhofer** [rubyforgood/casa#5293]
**Description**
The README and issue template (which created this issue) slack links are not working. 

#### Bitly Link
http://bit.ly/rubyforgood
![Z-2023-10-19 at 23 56 06@2x](https://github.com/rubyforgood/casa/assets/444766/9e2f02eb-5fdb-45e8-97ae-79973aa39b46)

#### Heroku Link
https://rubyforgood.herokuapp.com
![Z-2023-10-19 at 23 56 47@2x](https://github.com/rubyforgood/casa/assets/444766/9a97e38d-1ea1-430f-af33-7e9950e8c763)

**Existing file(s) needing changing**
- README
- This issue template

**New file(s) needing creating**
- N/A

### Questions? Join Slack!
I can't! 🤣 

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Tuesday 5-7pm Pacific), stakeholder news, and upcoming new issues.

**Comment by mattzollinhofer** [comment]
I'll happily take this as a first PR if you have new links for me to update!

**Comment by schoork** [comment]
@mattzollinhofer Looks like the heroku link isn't mentioned. Maybe replace it with the Slack link: http://bit.ly/rubyforgood

**Comment by mattzollinhofer** [comment]
@schoork: does that bitly link work for you? When I (or my coworker) go to the link it says it's expired. I _think_ someone needs to regenerate a new slack invite link, but I don't use slack too much so I'm not positive.

**Comment by schoork** [comment]
You're right. It doesn't in Incognito. Let me try to get you one.

**Comment by mattzollinhofer** [comment]
I was able to get invited to slack, so that was great!

But, unfortunately the public facing links I'm aware of (1. README and 2. GitHub issue/PR template) are not working at the moment. I'm not a very heavy Slack user and I'm just showing up around here so I hesitate to do this myself. 

I clicked around in slack and found the steps below, but I'm hesitant to just generate the link without someone's blessing. Also, I think any link we create will expire after 400 uses so you'll probably run into this again eventually, but it's the best I could find. Once we get an invite link we're happy with, we'll also need to get whoever generated the bit.ly rubyforgood link to update it or merely likely just pick a new bit.ly url.

Reference: https://slack.com/help/articles/201330256-Invite-new-members-to-your-workspace


<img width="500" alt="image" src="https://github.com/rubyforgood/casa/assets/444766/10c6d4b5-c141-45ec-a2f9-832f000e5b0d">
